src/server/app.js

```diff
@@ -3,13 +3,36 @@ const path = require("path");
 const express = require("express");
 const bodyParser = require("body-parser");
 const api_router = require("./routers/api_router");
+const util = require("util");
 
 const app = express();
 
 app.use(express.static(path.join(__dirname, "../../public")));
 app.use(bodyParser.json());
 app.use(express.static("public"));
 
+// API Router
 app.use("/api/v1/", api_router);
 
+// Validation Error Handler
+app.use((err, req, res, next) => {
+  if ((err.message = "validation error")) {
```

Comment:
Is this meant to be `===`?

Reply:
Nice spot. Will change this and add a test for it.